src/main/java/com/minecolonies/core/datalistener/ResearchListener.java

```diff
@@ -128,10 +128,10 @@ else if(effectJson.has(RESEARCH_SUBTITLE_PROP) && effectJson.get(RESEARCH_SUBTIT
                         }
                     }
                 }
-                // If no levels are defined, assume will go from zero to max level as a single action, and store on/off.
                 else
                 {
-                    category.add(5f);
+                    // By default no levels defined = 1 enabled, 0 disabled
+                    category.add(1);
```

Comment:
wonder if anything else was affected by this?

Reply:
wondered too and spend 30mins checking the usages, seems we just do > 0 at other places

Reply:
The default strength being 5 was intended for the building unlocks, because the strength specifies what level of building is unlocked.  So unless you added levels to all of those, they'd all break.

Reply:
added the levels^^